docs/gravitino-server-config.md

```diff
@@ -11,69 +11,69 @@ This software is licensed under the Apache License version 2."
 
 Gravitino supports several configurations:
 
-1. **Gravitino server configuration**: Used to start up Gravitino server.
+1. **Gravitino server configuration**: Used to start up the Gravitino server.
 2. **Gravitino catalog properties configuration**: Used to make default values for different catalogs.
-3. **Some other configurations**: Includes configurations such as HDFS configuration.
+3. **Some other configurations**: Includes HDFS and other configurations.
 
 ## Gravitino server configurations
 
-You can customize the Gravitino server by editing the configuration file `gravitino.conf` in the `conf` directory. The default values are sufficient for most use cases and don't need modification.
-It's strongly recommended to read the following sections to understand the configuration file and change the default values to suit your specific situation and usage scenario.
+You can customize the Gravitino server by editing the configuration file `gravitino.conf` in the `conf` directory. The default values are sufficient for most use cases.
+We strongly recommend that you read the following sections to understand the configuration file so you can change the default values to suit your specific situation and usage scenario.
```

Comment:
The Google document guidelines we are following https://developers.google.com/style suggest avoiding first person, but I think it's clear who we and you are here.